### DISCUSSION TITLE
如何将form.values转成普通对象？

### DISCUSSION THREAD

**Discussion by Alex-xd** [alibaba/formily#3459] [Q&A 2.x] [answered]
目前用JSON.parse(JSON.stringify(form.values))去做的，有点暴力。想问下有没有更好的方法？

**Comment by janryWang** [comment, marked as answer]
reactive toJS

**Reply by jiojio-jx** [reply]
刚刚尝试引用进来，报错了

**Reply by jiojio-jx** [reply]
![9C16860F356E56D27C42BB1916FE0E44](https://user-images.githubusercontent.com/54803764/205473236-f76d7879-5f9d-41b3-bc29-51cb63dc69e7.jpg)

**Reply by jiojio-jx** [reply]
异步加载包后、也不行
